DigitalOffice/Tests/TimeManagementServiceTests/Repositories/LeaveTimeRepositoryTests.cs

```diff
@@ -0,0 +1,88 @@
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Repositories;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Enums;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Repositories
+{
+    public class CreateWorkTimeTests
+    {
+        private TimeManagementDbContext dbContext;
+        private ILeaveTimeRepository repository;
+
+        private static readonly Guid worker1 = Guid.NewGuid();
+        private static readonly Guid worker2 = Guid.NewGuid();
+
+        private readonly DbLeaveTime testLeaveTime1 = new DbLeaveTime
+        {
+            Id = Guid.NewGuid(),
+            LeaveType = LeaveType.SickLeave,
+            Comment = "SickLeave",
+            StartTime = new DateTime(2020, 7, 5),
+            EndTime = new DateTime(2020, 7, 25),
+            WorkerUserId = worker1
+        };
+
+        private readonly DbLeaveTime testLeaveTime2 = new DbLeaveTime
+        {
+            Id = Guid.NewGuid(),
+            LeaveType = LeaveType.Training,
+            Comment = "SickLeave",
+            StartTime = new DateTime(2020, 7, 10),
+            EndTime = new DateTime(2020, 7, 20),
+            WorkerUserId = worker2
+        };
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            var dbOptions = new DbContextOptionsBuilder<TimeManagementDbContext>()
+                                    .UseInMemoryDatabase(databaseName: "InMemoryDatabase")
+                                    .Options;
+            dbContext = new TimeManagementDbContext(dbOptions);
+            repository = new LeaveTimeRepository(dbContext);
+        }
+
+        [TearDown]
+        public void Clear()
+        {
+            if (dbContext.Database.IsInMemory())
+            {
+                dbContext.Database.EnsureDeleted();
+            }
+        }
+
+        #region CreateLeaveTimeTests
+        [Test]
+        public void SuccessfulAddNewLeaveTimeInDb()
+        {
+            var guidOfNewLeaveTime = repository.CreateLeaveTime(testLeaveTime1);
+
+            Assert.AreEqual(testLeaveTime1.Id, guidOfNewLeaveTime);
+            Assert.NotNull(dbContext.LeaveTimes.Find(testLeaveTime1.Id));
+        }
+        #endregion
+
+        #region GetUserLeaveTimesTests
+        [Test]
+        public void CorrectlyReturnsLeaveTime()
+        {
+            dbContext.Add(testLeaveTime1);
+            dbContext.Add(testLeaveTime2);
+            dbContext.SaveChanges();
+
+            var leaveTimesOfWorker1 = repository.GetUserLeaveTimes(worker1);
+            Assert.AreEqual(leaveTimesOfWorker1.Count, 1);
+            Assert.IsTrue(leaveTimesOfWorker1.Contains(testLeaveTime1));
+
+            var leaveTimesOfWorker2 = repository.GetUserLeaveTimes(worker2);
+            Assert.AreEqual(leaveTimesOfWorker2.Count, 1);
+            Assert.IsTrue(leaveTimesOfWorker2.Contains(testLeaveTime2));
```

Comment:
I think, that methods of Assert must be next to each other

Reply:
Fixed.